applications/InkWhale_LaunchPad_using_Ink.md

```diff
@@ -0,0 +1,127 @@
+
+Phuong Hoang has been in sale and marketing industry for over a decade. She has been Sale and Marketing Manager/Director for many companies including: Honda Vietnam, Plex Cinama, Saga Media etc.
+
+All of our developers have at least 5-10 years experience in software development.
+
+### Team Code Repos
+
+- https://github.com/artzero-io
+- https://github.com/InkWhale-net
+
+### Team LinkedIn Profiles (if available)
+
+- https://www.linkedin.com/in/nad128668/
+- http://linkedin.com/in/phuong-phoebe-hoang-4b5888148
+- https://www.linkedin.com/in/tran-albert-469b6319a
+
+## Development Status :open_book:
+
+The Platform is under development already with Front-end Design at:
+https://www.figma.com/file/63xCCH71Oa8AfJpkK1wCO3/Ink-Whale?type=design&node-id=538-107&mode=design&t=jzPEaYJCUZLjUlDe-0
+
+Smart Contract Code:
+https://github.com/InkWhale-net/contracts/tree/nam-dev/contracts
+
+Back end Code:
+https://github.com/InkWhale-net/backend/tree/develop
+
+
+## Development Roadmap :nut_and_bolt:
+
+
+### Overview
+
+- **Total Estimated Duration:** 2-3 months
+- **Full-Time Equivalent (FTE):**  5 FTE
+- **Total Costs:** 20,000 USD
+
+### Milestone 1  — InkWhale - LaunchPad Platform using Ink! Smart Contract
+* **Estimated Duration:** 2-3 months
+* **Costs:** 20,000 USD
+
+
+| Number | Deliverable | Specification |
+| ------------- | ------------- | ------------- |
+| 0a.    | License            | Apache License 2.0                                           |
+| 0b.    | Documentation      | We will provide technical documents and user guides |
+| 0c.    | Testing Guide      | We will provide Unit Test for the Smart Contract and instructions how to perform the test |
+| 0d.    | Article/Tutorial   | We will write an article or tutorial that explains the work principle as part of the grant. |
+| 1. | Smart Contract Development | We will create 3 main contracts: LaunchPad generator as factory contract to initializer LaunchPad Contract; LaunchPad contract which has all features listed in the Project Details and a general PSP22 token contract for testing purposes|
+| 2. | Backend | We use nodejs and mongodb on AWS Services. The cronjob monitor update queue to make sure data in the database match with data on-chain; The API serves the frontend with following functions: interacting with IPFS network, get launchpad information, get token information, update launchpad information when needed |
+| 3. | Frontend | We will implement the Figma design as in https://www.figma.com/file/63xCCH71Oa8AfJpkK1wCO3/Ink-Whale?type=design&node-id=538-107&mode=design&t=jzPEaYJCUZLjUlDe-0 for the launchpad which contains 11 pages include: Project List; Project Creation (5 sub-pages); Project Details for Project Owner; Project Details for Public; Edit LaunchPad Information; Edit Whitelist and Update Phases;|
+
+
+## Future Plans
+
+
+## Additional Information :heavy_plus_sign:
```

Comment:
Please provide total cost. Is there only 1 milestone?

Reply:
yes we plan to do everything in 1 milestone